packages/ngspice/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: ngspice
+  version: "44.2"
+
+source:
+  url: https://downloads.sourceforge.net/project/ngspice/ng-spice-rework/44.2/ngspice-44.2.tar.gz
+  sha256: e7dadfb7bd5474fd22409c1e5a67acdec19f77e597df68e17c5549bc1390d7fd
```

Comment:
We don't prefer sourceforge URL's as it is often quite unstable. It would be nice if you can use some other URL's such as GitHub release.

Reply:
Unfortunately this project is primarily developed on Sourceforge, and indeed they move old versions into a folder, breaking the URL. There are some unofficial github mirrors but they don't publish releases to my knowledge.

You can see the conda recipe also uses this URL, and indeed if you try to run it you'll get a 404.

Would you prefer some tag from an unofficial mirror? eg
https://github.com/imr/ngspice/archive/refs/tags/ngspice-44.2.zip

Reply:
I think you can download the binary from Sourceforge, host it somewhere in your personal GitHub repository, and download it from there.